data/json/professions.json

```diff
@@ -3417,5 +3417,368 @@
+                "level": 1,
+                "name": "survival"
+            },
+            {
+                "level": 1,
+                "name": "fabrication"
+            },
+            {
+                "level": 1,
+                "name": "firstaid"
+            }
+        ],
+        "items": {
+            "both": [
+                "tshirt",
+                "jacket_light",
+                "pants_cargo",
+                "socks",
+                "boots",
+                "knit_scarf",
+                "backpack",
+                "granola",
+                "water_clean",
+                "scissors",
+                "magnifying_glass",
+                "flashlight",
+                "1st_aid"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "boy_shorts"
+            ]
+        }
+    },
+    {
+
+            "type": "profession",
+            "ident": "bionic_student",
+            "name": "Bionic Student",
+            "description": "Your parents were so obsessed with making sure you aced every test that they had you outfitted with bionics to make you smarter and never forget anything. And now, you are facing the most dire test yet, and once again you had better succeed, or else.",
+            "points": 3,
+            "CBMs":[
+                "bio_batteries",
+                "bio_power_storage",
+                "bio_int_enhancer",
+                "bio_memory"
+            ],
+            "items": {
+                "both": [
+                    "dress_shirt",
+                    "jacket_light",
+                    "pants",
+                    "socks",
+                    "dress_shoes",
+                    "skinny_tie",
+                    "tieclip",
+                    "fancy_sunglasses",
+                    "knit_scarf",
+                    "wristwatch",
+                    "mbag",
+                    "water_mineral",
+                    "pda",
+                    "cell_phone",
+                    "money_bundle"
+                ],
+                "male": [
+                    "briefs"
+                ],
+                "female": [
+                    "panties"
+                ]
+            }
+    },
+    {
+        "type": "profession",
+        "ident": "dodgeball_player",
+        "name": "Dodgeball Player",
+        "description": "You liked to play dodgeball, where failing to dodge the ball meant you were out. Now failing to dodge threatens your life. Don't slip up.",
+        "points": 1,
+        "skills": [
+            {
+                "level": 1,
+                "name": "dodge"
+            },
+            {
+                "level": 1,
+                "name": "throw"
+            }
+        ],
+        "items": {
+            "both": [
+                "tshirt",
+                "jacket_light",
+                "jeans",
+                "socks",
+                "lowtops",
+                "runner_bag",
```

Comment:
Runner bag is a pretty decent starting storage (2 less volume than the mbag).  IMO it doesn't *quite* warrant a second point but that's open to discussion.